src/System.Linq.Expressions/src/System/Linq/Expressions/ExpressionStringBuilder.cs

```diff
@@ -646,6 +646,8 @@ protected internal override Expression VisitUnary(UnaryExpression node)
                     Out("+");
                     break;
                 case ExpressionType.Quote:
+                case ExpressionType.PostIncrementAssign:
+                case ExpressionType.PostDecrementAssign:
```

Comment:
Right. Ensuring that we are handling all the cases would be good here.
The original code was likely not caring much about perf and was written at the time the nodes were added/removed. Now it is a stable set and a comprehensive switch would make more sense.